notes/stories/film/film-scifi-2000s.st.txt

```diff
@@ -2932,6 +2932,54 @@ biological computer [Dr. Yuhara explained that Mechagodzilla was installed with
 nuclear weapons [Sara stated that it was the use of the H-bomb the first attracted Godzilla to Japan.]
 
 
+movie: The Core (2003)
+======================
+
+:: Title
+The Core
+
+:: Description
+The Core is a 2003 American science fiction disaster film directed by Jon
+Amiel and starring Aaron Eckhart, Hilary Swank, Delroy Lindo, Stanley Tucci,
+Tchéky Karyo, DJ Qualls, Bruce Greenwood and Alfre Woodard.
+
+Synopsis: A team is sent on a mission to drill to the center of the Earth and
+set off a series of nuclear explosions in order to restart the rotation of the
+Earth's core.
+
+:: Date
+2003-03-28
+
+:: References
+https://en.wikipedia.org/wiki/The_Core
+
+:: Ratings
+2 <paul>
+
+:: Major Themes
+travel into the Earth [A team of scientists and NASA pilots journeyed down to the interface of Earth's outer and inner core in a special heat and pressure resistant vehicle.]
+government secrecy [The story concerns a secret United States government project to send a team of specialists to Earth's core to set off a series of nuclear explosions in a desperate bid to restart the planet's magnetic field.]
```

Comment:
I notice you use this a lot. I don't doubt it was a theme, but I wonder if it was really a major theme - was the secretness and the governmental orchestration really topical? Or was it just a minor background detail to otherwise tell a story about a group of people on an adventure?

Reply:
I often find myself having just this internal dialogue. But in this story at least it is justifiable as major, since the government coerced a notorious hacker into overseeing that word of the project was kept secret from the public, and we saw him busy at work doing just so. And there are other details, like how in the end it was a sensation when news of the secret project was leaked on the Internet, and so on.